packages/transport-http/test/HttpTransportMethods.test.js

```diff
@@ -0,0 +1,23 @@
+import HttpTransport from '../src';
+
+describe('HttpTransport Methods', () => {
+    it('test getHttpMethodAndUrlFromQueueName', () => {
+        const transport = new HttpTransport();
+        expect(transport.getHttpMethodAndUrlFromQueueName('a:b:c:get')).toEqual({
```

Comment:
that will fail

Reply:
I think this is fixed please review again